httpbakery/client_test.go

```diff
@@ -34,6 +33,8 @@ type ClientSuite struct {
 
 var _ = gc.Suite(&ClientSuite{})
 
+var macaroonCurrentVersion = bakery.MacaroonVersion(bakery.LatestVersion)
```

Comment:
Are the Macaroon version and bakery version always going to be in lockstep? in which case why have two versions?